docs/tests.md

```diff
@@ -6,6 +6,7 @@
 * Live run the test suite with `npm run cypress:open`
 * Run the tests in a headless environment with `npm run cypress:cli`
 * Travis will invoke `npm run cypress:travis` to run the tests
+* If you're having issues with port 8080 being already in use run this command: `kill -9 $(lsof -t -i:8080)`
```

Comment:
these instructions need to say that the webserver that cypress runs needs to be done over a build. That OR include the `npm run build` to the cypress script.

same goes for the accessibility tests.